### DISCUSSION TITLE
Meeting 2024-10-08

### DISCUSSION THREAD

**Discussion by jamesmunns** [rust-embedded/wg#798] [General]
This is the agenda for the next meeting. Please add anything you'd like to discuss below!

* Meetings: Tuesday 8pm Europe/Berlin time
* [Join the Matrix chat](https://matrix.to/#/#rust-embedded:matrix.org)
* [Nominated issues](https://github.com/search?q=org%3Arust-embedded+label%3Anominated+is%3Aopen&type=Issues)
* [IRC/Matrix logs](https://libera.irclog.whitequark.org/rust-embedded/2024-10-08)

**Comment by rmsyn** [comment]
In the [riscv](https://github.com/rust-embedded/riscv) project, we have two open PRs ready for wider review before merging:

- [Support for non-standard exception and interrupts](https://github.com/rust-embedded/riscv/pull/223)
- [Add CSR-defining macros](https://github.com/rust-embedded/riscv/pull/219)
  - introduces [paste](https://docs.rs/paste) as a re-export (public, but not intended for direct external use)

**Comment by thejpster** [comment]
https://github.com/rust-lang/rust/pull/130987

Also there's a lot of chat on Zulip about Arm targets and -C flags that is worth catching up on.

**Comment by d3zd3z** [comment]
Just a brief mention that rust application support in Zephyr has merged into the Zephyr tree.

**Comment by thejpster** [comment]
Plus compiler-builtins was broken, causing havoc for SPARC and WASM. Some out of tree testing might be useful.

**Comment by BartMassey** [comment]
Reminder: Next book sprint for the Discovery MB2 book will be on October 26, 17:00-21:00 CEST, 15:00-19:00 UTC, 8:00AM-12:00PM PST. Join if you're interested in helping to write the new discovery book! Give a thumbs up to [this comment](https://github.com/rust-embedded/wg/discussions/797#discussioncomment-10812164) to express your interest in joining.

**Comment by thejpster** [comment]
https://github.com/rust-lang/rust/issues/131164

The linker broke on nightly and barely anyone noticed.

**Reply by jannic** [reply]
(Only affects certain situations - so it's not like nobody tests nighty / beta.)
